dropwizard-unix-socket/pom.xml

```diff
@@ -0,0 +1,59 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>io.dropwizard</groupId>
+        <artifactId>dropwizard-parent</artifactId>
+        <version>4.0.3-SNAPSHOT</version>
+        <relativePath>../dropwizard-parent</relativePath>
+    </parent>
+
+    <artifactId>dropwizard-unix-socket</artifactId>
+    <name>Dropwizard Unix Socket Support</name>
+
+    <properties>
+        <module.name>io.dropwizard.dropwizard.unixsocket</module.name>
+        <maven.compiler.release>16</maven.compiler.release>
```

Comment:
That one's not good. If one builds the project with Java 11 - which currently is the minimum required version - this will cause problems.

Reply:
I'm not very experienced with maintaining libraries. Any chance we can make this package optional and require java 16 without breaking dropwizard-core's java 11 support?